src/measure_theory/function/conditional_expectation.lean

```diff
@@ -328,6 +330,24 @@ instance [hm : fact (m ≤ m0)] [complete_space F] [hp : fact (1 ≤ p)] :
   complete_space (Lp_meas F 𝕜 m p μ) :=
 by { rw (Lp_meas_to_Lp_trim_lie F 𝕜 p μ hm.elim).to_isometric.complete_space_iff, apply_instance, }
 
+include 𝕜
+variables (𝕜)
+lemma is_complete_ae_measurable' [hp : fact (1 ≤ p)] [complete_space F] (hm : m ≤ m0) :
```

Comment:
There is something a little bit strange going on here: the statement does not involve `𝕜` at all, but still it is needed for the proof. This could probably be fixed by using instead of the submodule `Lp_meas F 𝕜 m p μ` an `add_subgroup`, but I am not sure it is worth it. In all the applications, one will have real or complex vector spaces, right? Still, it might make sense to mention this in some comments.

Reply:
I changed the whole proof of completeness to use an `add_subgroup` instead of a `submodule`. The submodule is still defined and used elsewhere in the file. The subgroup is used only to prove completeness.

The diff looks big, but I mostly added `_subgroup` to every mention of `Lp_meas` in that section.

Reply:
Thanks!